modules/tfm/zephyr/Kconfig

```diff
@@ -74,8 +74,8 @@ config TFM_UART1_TXD_PIN
 	default 25 if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
 	default 4294967295 # 0xffffffff i.e. disconnected
 	help
-	  The GPIO pin the TFM build system will use for UART0 TXD pin.
-	  UART0 is used by TF-M as the log output for the secure image.
+	  The GPIO pin the TFM build system will use for UART1 TXD pin.
+	  UART1 is used by TF-M as the log output for the secure image.
 
 config TFM_UART1_RXD_PIN
 	int "UART0 TXD pin"
```

Comment:
I noticed all prompts say `int "UART0 TXD pin"` as well

Reply:
Yes, thanks. I even wrote that in the commit message but forgot to update :P